LabController/src/bkr/labcontroller/expire_distros.py

```diff
@@ -93,7 +93,7 @@ def check_all_trees(ignore_errors=False,
                         print('{0} is missing [Distro Tree ID {1}]'.format(
```

Comment:
Should this be " instead of ' in this section?  Looks like this change wasn't done in this file.

Reply:
I think it's covered in another commit.